cloudinit/sources/DataSourceAzure.py

```diff
@@ -1436,9 +1435,15 @@ def _get_metadata_from_imds(retries):
         report_diagnostic_event(msg)
         LOG.debug(msg)
         return {}
+    try:
+        import json
+        json_decode_error = json.decoder.JSONDecodeError
+    except ImportError, AttributeError:
```

Comment:
I think we can simplify this a little:

```suggestion
    try:
        from json.decoder import JSONDecodeError
    except ImportError:
```